bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -28,6 +28,22 @@ limitations under the License.
     <mainClass>com.google.cloud.bigtable.beam.sequencefiles.Main</mainClass>
   </properties>
 
+  <!-- Adding this to resolve version conflict within beam sdk-->
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-api</artifactId>
+        <version>1.29.0</version>
```

Comment:
please replace this with a variable; `beam-grpc.version`

Reply:
Done.